public/demo.html

```diff
@@ -0,0 +1,429 @@
+
```

Comment:
Just a demo for someone to preview tile using leaflet and test how the tile layer can interact with a background.   The index.html provided by core tessera was OK but our guys wanted to see tiles on top of basemap and potentially tile with multiple layers in it.

Will remove in next PR as it is not core tessera.   Sorry , this was slipped in.